contributing.md

```diff
@@ -31,12 +31,36 @@ Please open a GitHub issue to suggest a new feature, contribute code, or let us
 🛠 I am ready to contribute 
 ---
 
-- For open tasks in this repository, please see the [Issues section](https://github.com/alan-turing-institute/scivision/issues).
-- Raise mistakes, errors or missing information on this repository by opening a [Pull Request](https://github.com/alan-turing-institute/scivision/pulls)
+- For open tasks in this repository, please see the open [Issues](https://github.com/alan-turing-institute/scivision/issues).
+
+- To contribute to this repository (including bugfixes and new code features, as well as documentation, fixes to typos, broken links), open a [Pull Request](https://github.com/alan-turing-institute/scivision/pulls).
+
+- Our basic workflow is [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow).  In particular:
+  - The branch `main` always contain the latest working version of the package.
+  - Pull Requests should normally have `main` as their base branch (this is the default).
+  - A Pull Request doesn't have to represent finished work. Pull Requests are welcome at an early stage of work on a contribution, so others can watch or give feedback, or as a place to discuss the work in progress---you can always add more commits to the same branch later, and they will be included too. If it is not ready to merge, include "WIP" (Work in Progress) in the subject line, or select "Create Draft Pull Request" when opening it.
+  - For a Pull Request to be ready to merge:
+    - All of the automated tests should pass
+    - It should add tests of any new functionality
+  - A code review can be requested at any time
+  - Pull requests are [squash merged]() to `main`
+
+- More information:
   - Read details on [how to open a Pull Request](https://opensource.guide/how-to-contribute/#opening-a-pull-request)
-  - Submit trivial fixes (for example, a typo, a broken link or an obvious error)
-  - Start work on a contribution that is already listed as an issue or something you’ve already discussed
-  - A Pull Request doesn’t have to represent finished work. It’s usually better to open a Pull Request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. You can always add more commits later.
+
+### Releases
+
+This section is for developers making a release: Most contributors
+should instead read the notes on opening a Pull Request above.
+
+We periodically make releases of the software on [PyPI](https://pypi.org/project/scivision/).
+
+To make a new release:
+  - Make a new branch the release (e.g. named `release/1.1.0`). Usually this will be branch off the current `main`.
+  - Make sure that the version number is bumped in `setup.py`
+  - When ready, request a review and merge into main
+  - Make a GitHub [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) (which will also make a tag)
+  - (add/link to PyPI instructions here)
```

Comment:
I have added instructions in the [ds4s repo](https://github.com/alan-turing-institute/DS4S-project-management/tree/main/scivision) - @ots22 they could be put here instead, I only didn't because I guess an open contributor would not be a maintainer and therefore would be unable to do this, so put in the private repo for now. Perhaps we should add it here as an extra step like:

- Reach out to one of the following pypi maintainers...
- Ask them to upload your release (or request access)

Reply:
Alternatively, if we can find a solution to #86 then problem solved

Reply:
Ah right, thanks, didn't notice those! Maybe here, or somewhere else in this repo instead? They won't be relevant for everyone of course, but probably the first place somebody would look for them (myself included!) Having the release process documented openly also seems like a good thing.

Reply:
Moved the release instructions to the docs which seems like the right place, and linked the contributing guide from there as well.

Reply:
I added the release info to the README in the PR I just merged, but feel free to move it to where you've put it here if you prefer